src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackActionsBlock.php

```diff
@@ -24,16 +24,26 @@ public function __construct()
     /**
      * @return $this
      */
-    public function button(string $text, string $url, ?string $style = null): static
+    public function button(string $text, ?string $url = null, ?string $style = null, ?string $value = null): static
```

Comment:
I'm not sure this change makes sense. Buttons defined this way are legacy anyway?
What would the value be used for, for buttons?